ethstats/ethstats.go

```diff
@@ -621,7 +626,7 @@ func (s *Service) reportLatency(conn *websocket.Conn, sendCh chan *StatsPayload)
 		// Ping timeout, abort
 		return errors.New("ping timed out")
 	}
-	latency := strconv.Itoa(int((time.Since(start) / time.Duration(2)).Nanoseconds() / 1000000))
+	latency := int((time.Since(start) / time.Duration(2)).Nanoseconds() / 1000000)
```

Comment:
Is this what Primus expects?

Reply:
no this is what celostats expects, this is the payload of the latency response. The value is converted to number on arrival at celostats to be used as number in latency calculation. To save this conversion it makes sense to transfer it as number from the start